src/OpenSilver.TemplateWizards/AppCustomizationWindow/ThemeCollectionView.xaml

```diff
@@ -84,10 +84,10 @@
                                         </VisualStateManager.VisualStateGroups>
 
                                         <Border x:Name="mouseOverIndicator" Opacity="0" Background="White" IsHitTestVisible="False" BorderThickness="{TemplateBinding BorderThickness}" CornerRadius="10" BorderBrush="Transparent"/>
-                                        <Grid >
+                                        <Grid>
```

Comment:
It makes it possible to select a platform when clicking between the icon and its title
```suggestion
                                        <Grid Background="Transparent">
```